test/testdata/packager/layer/invalid_option/__package.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+# typed: strict
+# enable-packager: true
+# packager-layers: a, b
```

Comment:
out of curiosity, where does the code ignore the leading space in `, `?

Reply:
The `trim_string` call here: https://github.com/sorbet/sorbet/blob/master/test/helpers/position_assertions.cc#L2331